frontend/yarn.lock

```diff
@@ -8023,6 +8087,21 @@ kind-of@^6.0.0, kind-of@^6.0.2:
   version "6.0.2"
   resolved "https://registry.yarnpkg.com/kind-of/-/kind-of-6.0.2.tgz#01146b36a6218e64e58f3a8d66de5d7fc6f6d051"
 
+kubevirt-web-ui-components@^0.1.34-4:
```

Comment:
Can we consider removing this in this PR? We're only using `getName` and `getNamespace` currently. Those are trivial to implement.

Having this dependency defeats the purpose of the monorepo. Is it feasible to simply bring the components in to console directly as we add the capabilities? (I don't know the code well enough to say.)

Reply:
Sam, this is just initial PR with bare minimum functionality to enable transfer of the rest in follow-ups. I guess 80% of the planed functionality is recently in kubevirt-web-ui.

As mentioned before: we plan to deprecate `kubevirt-web-ui` asap and so remove it even from dependencies here.

But doing it now would cause us a lot of troubles in case we do not achieve full feature parity in 4.2 timeframe (considering recent progress, this is valid concern).

So if possible, it will be huge relieve for us to keep it and go safely feature-by-feature.

Reply:
The whole kubevirt-web-ui-components will be moved into this monorepo, but only as a last step. I would like to avoid duplicating the code as part of this effort and having the same stuff on two places. We still need to maintain the fork and its releases so it is important to have it on one place only and if we make fix there, not to have to create a new fix also here.

Reply:
Until the transition from our fork to Console plugins is complete (in terms of feature parity), it's more practical to use `kubevirt-web-ui-components` as an external library. As @mareklibra mentioned, there's always the chance of not achieving full parity in 4.2 timeframe.

Once our fork is discontinued, we can
- either move `kubevirt-web-ui-components` as-is under e.g. `packages/kubevirt-components` (which isn't ideal, since it's plain JS and its code & deps might need revisiting, like `react-measure` mentioned above)
- or we can create an empty e.g. `packages/kubevirt-components` and gradually bring in the necessary bits (revisiting & migrating them to TS in the process) and eventually remove the `kubevirt-web-ui-components` dependency

Personally, I favor the second approach.